GoogleMapsUtils.xcodeproj/project.pbxproj

```diff
@@ -1524,6 +1542,8 @@
 				);
 				PRODUCT_BUNDLE_IDENTIFIER = com.google.gmsutils.DevApp;
 				PRODUCT_NAME = "$(TARGET_NAME)";
+				SWIFT_OBJC_BRIDGING_HEADER = "./App/DevApp-Bridging-Header.h";
```

Comment:
The `DevApp` target does not compile/run for me. This file needs to be checked in.

![image](https://user-images.githubusercontent.com/463186/90798023-3a6f5c00-e2c6-11ea-9456-66b13a5aa18f.png)

Reply:
That's weird... I don't believe I touched a bridging header after we realized it wasn't needed. Was this bridging header something I accidentally created or was it there in the first place? I'm a little bit confused; sometimes the Xcode project pane files don't match up with files in the Finder for some reason.
Sorry about that!!

Reply:
No worries, it takes time to get used to Xcode. I find it annoying that the files in the file directory don't match the structure displayed in Xcode. :/ 

You must have created the bridging header file when you added the new .swift file. If your new .swift file doesn't have `DevApp` as a target (I think it only targets `GoogleMapsUtils`?), you can actually remove the reference to the bridging header file altogether.

Reply:
Ok I've removed those references in the pbxproj file! I don't think they were ever removed from the initial attempt (using bridging header files); weird how git checkout a clean commit didn't remove those lines.